actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -313,8 +313,8 @@ private void courseProgressMetricsReport(Request actorMessage) {
             "CourseMetricsActor:courseProgressMetricsReport: assessmentBatchResult="
                     + assessmentBatchResult,
             LoggerEnum.INFO.name());
-    if (isNotNull(assessmentBatchResult) || assessmentBatchResult.size() != 0) {
-      List<Map<String, Object>> content = (List<Map<String, Object>>) assessmentBatchResult.get(JsonKey.CONTENT);
+    List<Map<String, Object>> content = (List<Map<String, Object>>) assessmentBatchResult.get(JsonKey.CONTENT);
+    if (isNotNull(content) && content.size() > 0) {
```

Comment:
please use null and empty check as below
if (MapUtils.isNotEmpty(assessmentBatchResult)
        && CollectionUtils.isNotEmpty((List<Map<String, Object>>) assessmentBatchResult.get(JsonKey.CONTENT)))